packages/spotlight/bin/run.js

```diff
@@ -20,55 +20,44 @@ const homeDir = process.env.HOME || process.env.USERPROFILE;
 setContext("CLI", {
   sea: sea.isSea(),
   // TODO: Be less naive with path obscuring
-  argv: process.argv.map((arg) => arg.replace(homeDir, "~")),
+  argv: process.argv.map(arg => arg.replace(homeDir, "~")),
 });
 
 const withTracing =
   (fn, spanArgs = {}) =>
   (...args) =>
-    startSpan({ name: fn.name, attributes: { args }, ...spanArgs }, () =>
-      fn(...args)
-    );
+    startSpan({ name: fn.name, attributes: { args }, ...spanArgs }, () => fn(...args));
 
 const readAsset = withTracing(
   sea.isSea()
-    ? (name) => Buffer.from(sea.getRawAsset(name))
+    ? name => Buffer.from(sea.getRawAsset(name))
     : (() => {
-        const ASSET_DIR = join(
-          fileURLToPath(import.meta.url),
-          "../../dist/overlay/"
-        );
+        const ASSET_DIR = join(fileURLToPath(import.meta.url), "../../dist/overlay/");
 
-        return (name) => readFileSync(join(ASSET_DIR, name));
+        return name => readFileSync(join(ASSET_DIR, name));
       })(),
-  { name: "readAsset", op: "cli.asset.read" }
+  { name: "readAsset", op: "cli.asset.read" },
 );
 
 startSpan({ name: "Spotlight CLI", op: "cli" }, async () => {
-  await startSpan(
-    { name: "Setup Sidecar", op: "cli.setup.sidecar" },
-    async () => {
-      const MANIFEST_NAME = "manifest.json";
-      const ENTRY_POINT_NAME = "src/index.html";
-      const filesToServe = Object.create(null);
+  await startSpan({ name: "Setup Sidecar", op: "cli.setup.sidecar" }, async () => {
+    const MANIFEST_NAME = "manifest.json";
+    const ENTRY_POINT_NAME = "src/index.html";
+    const filesToServe = Object.create(null);
 
-      startSpan(
-        { name: "Setup Server Assets", op: "cli.setup.sidecar.assets" },
-        () => {
-          // Following the guide here: https://vite.dev/guide/backend-integration.html
-          const manifest = JSON.parse(readAsset(MANIFEST_NAME));
-          filesToServe[ENTRY_POINT_NAME] = readAsset(ENTRY_POINT_NAME);
-          const entries = Object.values(manifest);
-          for (const entry of entries) {
-            filesToServe[entry.file] = readAsset(entry.file);
-          }
-        }
-      );
+    startSpan({ name: "Setup Server Assets", op: "cli.setup.sidecar.assets" }, () => {
+      // Following the guide here: https://vite.dev/guide/backend-integration.html
+      const manifest = JSON.parse(readAsset(MANIFEST_NAME));
+      filesToServe[ENTRY_POINT_NAME] = readAsset(ENTRY_POINT_NAME);
+      const entries = Object.values(manifest);
+      for (const entry of entries) {
+        filesToServe[entry.file] = readAsset(entry.file);
+      }
+    });
```

Comment:
this is the kind of ting that pisses me off. this is not a change, just the formatting, but the ai decided to change it

Reply:
I think it was reformatting, that's why I left it in.

Reply:
then perfect.